src/main/java/frc/robot/commands/shooter/FullAutoShooterAssembly.java

```diff
@@ -0,0 +1,196 @@
+package frc.robot.commands.shooter;
```

Comment:
I feel like you accidentally merged in the Shooter Assembly branch or made this branch from that one. Clean up the branch and remove the files associated with the shooter assembly.